dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EventResource.java

```diff
@@ -99,7 +102,9 @@ public List<Event> getEvents(@Parameter(hidden = true) @ApiParam(hidden = true)
                                  @NotNull @QueryParam("eventSearchType") EventSearchType eventSearchType,
                                  @Min(1) @Max(MAX_LIMIT) @DefaultValue(PAGINATION_DEFAULT_STRING) @ApiParam(defaultValue = PAGINATION_DEFAULT_STRING, allowableValues = PAGINATION_RANGE) @Parameter(schema = @Schema(maximum = "100", minimum = "1")) @QueryParam("limit") Integer limit,
                                  @QueryParam("offset") @DefaultValue("0") Integer offset) {
+        LOG.error("A");
```

Comment:
logging clean-up

Reply:
> Logging clean up comment seems applicable -- I know the PR description says it's a draft, but it's not a draft PR, unclear if that changed since the PR was first created, so I'm leaving the comment anyway. :)

Woops, yeah, re not being a draft PR, I musta goofed and pressed the wrong button.  I've pushed an attempt at the production version, it's pretty similar to what was already here minus the LOG statements...